File: Grupo_Rac/Grupo_Rac.Common/Models/CiudadViewModel.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Grupo_Rac.Common.Models
{
    public class CiudadViewModel
    {
        public string Ciu_Id { get; set; }
        public string? Dep_Id { get; set; }
        public string Ciu_Descripcion { get; set; }
        public int? Ciu_Creacion { get; set; }
        public int? Ciu_Modifica { get; set; }
        public DateTime Ciu_Fecha_Creacion { get; set; }
        public DateTime? Ciu_Fecha_Modifica { get; set; }
        public bool? Ciu_Estado { get; set; }
        [NotMapped]
        public string FechaModificacion { get; set; }
        [NotMapped]
        public string FechaCreacion { get; set; }
        [NotMapped]
        public string UsuarioCreacion { get; set; }

        [NotMapped]
        public string UsuarioModificacion { get; set; }
    }
}

File: sakai-ng-master/src/app/Models/CiudadViewModel.ts

export class Municipio{
    ciu_Id?:String;
    ciu_Descripcion?:String;
    departamento?:String;
}


export class MunicipioEnviar {
    Ciu_Id: string;
    Ciu_Descripcion: string;
    Dep_Id: string;
}

export class Fill {
    ciu_Id: string;
    ciu_Descripcion: string;
    dep_Id: string;
    departamento: string;
    usuarioCreacion: string;
    usuarioModificacion: string;
    fechaCreacion : string;
    fechaModificacion : string;
}

export class dropMunicipio{
    value?:String;
    text?:String;
}
